client-js/app/elements/dir-item.html

```diff
@@ -16,6 +16,9 @@
         dirIcon: {
           type: String,
           computed: 'assignIcon(kind)'
+        },
+        link: {
```

Comment:
It could. But the examples I've seen all separate the code like this. I don't know if it's according to a style guide or just to make the examples easier to read.